tests/tracer/test_ext.py

```diff
@@ -21,7 +25,154 @@ def _updateenv(monkeypatch, env):
         monkeypatch.setenv(str(k), str(v))
 
 
+@pytest.fixture
+def git_repo(tmpdir):
+    cwd = str(tmpdir)
```

Comment:
does this need to be a string?

Reply:
Yup, `tmpdir` actually returns a `localpath` that will break if not converted into a string in py2 😢

Reply:
what gives an error with it being a `localpath`?

Reply:
It leads to a `TypeError` like discussed in this issue: https://github.com/pypa/setuptools_scm/issues/55